### ISSUE TITLE
We need to know if the user is a team member or delegate

### ISSUE THREAD

**Issue by jfly** [jfly/wca-elections#1]
Right now, the WCA [`/me`](https://github.com/cubing/worldcubeassociation.org/blob/273d7bb75861778f7178c412d6c2dae86c787bbc/WcaOnRails/app/controllers/api/v0/api_controller.rb#L10-L14) route doesn't expose this information:

```
~ @kaladin> curl -H "Authorization: Bearer ...." https://www.worldcubeassociation.org/api/v0/me
{
  "me": {
    "class": "user",
    "url": "/results/p.php?i=2005FLEI01",
    "id": 8,
    "wca_id": "2005FLEI01",
    "name": "Jeremy Fleischman",
    "gender": "m",
    "country_iso2": "US",
    "created_at": "2010-12-29T22:25:01.000Z",
    "updated_at": "2016-05-18T15:40:24.000Z",
    "avatar": {
      "url": "https://d1qsrrpnlo9sni.cloudfront.net/uploads/user/avatar/2005FLEI01/1441167621.jpg",
      "thumb_url": "https://d1qsrrpnlo9sni.cloudfront.net/uploads/user/avatar/2005FLEI01/1441167621_thumb.jpg",
      "is_default": false
    }
  }
}
```

**Comment by pedrosino** [comment]
I don't know anything about /me, but can't you use current_user.delegate_status ?

Or maybe export a list of the current delegates and import to the other app as a team table? (Something we should probably do on the real db sometime)

**Comment by jfly** [comment]
This was fixed in 27d8d19a273462ab3898b5b1b2d029ed2d41be6d.
